tools/scxtop/src/bpf/main.bpf.c

```diff
@@ -677,11 +677,10 @@ int BPF_PROG(on_ipi_send_cpu, u32 cpu, void *callsite, void *callback)
 	return 0;
 }
 
-SEC("tp_bpf/gpu_memory_total")
+SEC("?tp_bpf/gpu_memory_total")
```

Comment:
note: I think there is a typo (my bad), should not it be tp_b**t**f/gpu_memory_total ?

Reply:
Should be good now, thanks for catching!

Reply:
@devnexen I think the tracepoint is `gpu_mem_total` from the [definition](https://web.git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/gpu/trace/trace_gpu_mem.c#n13).

Reply:
correct